src/testRunner/unittests/tscWatch/watchEnvironment.ts

```diff
@@ -497,5 +497,161 @@ namespace ts.tscWatch {
                 verifyWorker("-extendedDiagnostics");
             });
         });
+
+        describe("handles watch for extended configs", () => {

```

Comment:
This isnt watch environment test. It doesnt need test with watch options. Just ensuring extended files are watched and program gets updated. The test should go into the `programUpdates.ts`